src/core/manager.c

```diff
@@ -2527,16 +2528,19 @@ static int manager_dispatch_signal_fd(sd_event_source *source, int fd, uint32_t
 
                 case 26:
                 case 29: /* compatibility: used to be mapped to LOG_TARGET_SYSLOG_OR_KMSG */
+                        m->log_target_overridden = true;
                         log_set_target(LOG_TARGET_JOURNAL_OR_KMSG);
                         log_notice("Setting log target to journal-or-kmsg.");
```

Comment:
i figure this one should now revert to the default again, as LOG_TARGET_JOURNAL_OR_KMSG is the default for PID 1

Reply:
Agreed.